src/Contrib/OtlpGrpc/Exporter.php

```diff
@@ -116,7 +116,17 @@ protected function doExport(iterable $spans): int
             'resource_spans' => $resourceSpans,
         ]);
 
+        // @var \Opentelemetry\Proto\Collector\Trace\V1\ExportTraceServiceResponse|null $response
         [$response, $status] = $this->client->Export($request)->wait();
+        if ($response && $response->hasPartialSuccess()) {
```

Comment:
Updated to handle full/partial/warnings over gRPC.

Reply:
I think this can be simplified:
```php
if ($response && $response->hasPartialSuccess()) {
    if ($response->getPartialSuccess()->getRejectedSpans()) {
        // log error & return failure
    }
    if ($response->getPartialSuccess()->getErrorMessage()) {
        // log warning
    }
}
```